packages/ontario-frontend-cli/bin/ontario-create-app.js

```diff
@@ -10,77 +12,99 @@ const program = new Command();
 
 /**
  * readPackageJson() is an async function that is used to read the contents of the package.json file.
- * 
- * These contents are checked against in several places in the below code, therefore the all of the 
+ *
+ * These contents are checked against in several places in the below code, therefore the all of the
  * program commands should be wrapped in a self-executing async function.
- * 
+ *
  * Without this async wrapper function, the program will exit unexpectedly after user input.
- */ 
-(
-  async () => {
-    const packageJson = await readPackageJson(ROOT_DIR);
+ */
+(async () => {
+  const packageJson = await readPackageJson(ROOT_DIR);
 
-    program
-      .name('ontario-create-app')
-      .version(packageJson.version, '-v, --version', 'Output the current version')
-      .description('Create a new Ontario Frontend project')
-      .option(
-        '--projectName <name>',
-        'Specify the project name (lowercase, hyphens, underscores allowed)',
-        (value) => {
-          // Leverage the validation used with Commander for prompted questions
-          // The validation result is either True if the value is valid, or
-          // it is a string representing the error and remedy
-          const validationResult = validFileName(value);
-          if (validationResult !== true) {
-            logger.error('Invalid project name - ',validationResult);
-            process.exit(1);
-          }
-          logger.info('Using project name: ', value);
-          return value;
+  program
+    .name('ontario-create-app')
+    .version(packageJson.version, '-v, --version', 'Output the current version')
+    .description('Create a new Ontario Frontend project')
+    .option(
+      '--projectName <name>',
+      'Specify the project name (lowercase, hyphens, underscores allowed)',
```

Comment:
I think comment should match the ones below for enName and frName (alphanumeric, hyphens, underscores allowed) although lowercase may still have some value I don't think uppercase characters currently work

Reply:
Similar messaging is being used when the questions themselves are asked to the user in
`packages/ontario-frontend-cli/core/questions/languagePageQuestions.js` and
`packages/ontario-frontend-cli/core/questions/projectNameQuestion.js`

Maybe we can standardize the messaging across files?

Reply:
Standardized the parentheses to reflect the current regex - lowercase, alphanumeric, hyphens, underscores. 

languagePageQuestions requires "for URL path" because the value chosen here has implications on paths.